app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -34,6 +34,7 @@ def tab(*args)
           link = link_to_with_icon(options[:icon], titleized_label, destination_url)
           css_classes << 'tab-with-icon'
         else
+          titleized_label = raw("<span class='text'>#{titleized_label}</span>")
```

Comment:
I'd like to suggest a way DRY this format which is shared with `link_to_with_icon`, but can't think of a way... 🤔